ur_gazebo/urdf/ur_ftbasic.xacro

```diff
@@ -0,0 +1,126 @@
+
+  <!--
+     TODO:  add ${prefix} to all following link, joint, and topic names, and resolve
+   -->
+  <!--
+    Attach the Gazebo model to Gazebo's world frame.
+
+    Note: if you're looking to integrate a UR into a larger scene and need
+    to add EEFs or other parts, DO NOT change this file or the 'world' link
+    here. Create a NEW xacro instead and decide whether you need to add
+    a 'world' link there.
+  -->
+  <link name="world" />
+  <joint name="world_joint" type="fixed">
+    <parent link="world" />
+    <child link="base_link" />
+    <origin xyz="0 0 0" rpy="0 0 0" />
+  </joint>
+
+  <!-- 
+    Force torque sensor link, joint, and Gazebo definition
+    TODO:  parameterize FT sensor properties, e.g. choose Optoforce or OnRobotics by YAML 
+  -->
+
+  <link name="ft_sensor_link">
+    <visual>
+      <origin rpy="0 0 0" xyz="0 0 0.025" />
+      <geometry>
+        <cylinder radius="0.035" length="0.05" />
+      </geometry>
+      <material name="LightGrey">
+        <color rgba="0.7 0.7 0.7 1.0" />
+      </material>
+    </visual>
+    <collision>
+      <origin rpy="0 0 0" xyz="0 0 0.025" />
+      <geometry>
+        <cylinder radius="0.035" length="0.05" />
+      </geometry>
+    </collision>
+    <inertial>
+      <mass value="0.100" />
+      <origin rpy="0 0 0" xyz="0.0 0.0 0.025" />
+      <inertia ixx="9.890410052167731e-05" ixy="0.0" ixz="0.0" iyy="9.890410052167731e-05" iyz="0.0" izz="0.0001321171875" />
+    </inertial>
+  </link>
+
+  <!-- Use revolute joint as Gazebo's internal URDF to SRDF convertion lumps fixed links in parent -->
+  <joint name="ft_sensor_joint" type="revolute">
+    <parent link="wrist_3_link" />
+    <child link="ft_sensor_link" />
+    <origin xyz="0 0 0.001" rpy="0 0 0" />
+    <axis xyz="0 0 1" />
+    <limit lower="0" upper="0" effort="0" velocity="0" />
+    <dynamics damping="0" friction="0" />
+  </joint>
+
+  <!-- Disable collision with previous link -->
+  <disable_collisions link1="ft_sensor_link" link2="wrist_3_link" reason="Adjacent" />
+
+  <!-- Enable Gazebo sensor on joint -->
+  <gazebo reference="ft_sensor_joint">
+    <pose>0 0 0 0 0 0</pose>
+    <sensor name="ft_sensor" type="force_torque">
+      <always_on>true</always_on>
+      <update_rate>10.0</update_rate>
+      <visualize>true</visualize>
+      <force_torque>
+        <frame>sensor</frame>
+        <measure_direction>child_to_parent</measure_direction>
+      </force_torque>
+    </sensor>
+  </gazebo>
+
+  <!-- Use gazebo_ros_pkgs ft_sensor library to publish ft_sensor_topic by referencing wrist_3-ft_sensor_joint -->
+  <gazebo>
+    <plugin name="ft_sensor_plugin" filename="libgazebo_ros_ft_sensor.so">
+      <topicName>ft_sensor/ft_sensor_topic</topicName>
+      <jointName>wrist_3-ft_sensor_joint</jointName>
```

Comment:
Is it the correct joint name?

Reply:
I confirm that changing it makes the rostopic available. Previously it was not in my case. Please double check

Reply:
Yes sorry, I will be more thorough